scripts/merge-measures-data.js

```diff
@@ -47,9 +47,70 @@ function mergeQpp(qppJson) {
       if (performanceRateInfo.metricType) {
         qppJson[index].metricType = performanceRateInfo.metricType;
       }
+
     }
   });
 
   console.error('did not find measure details for the following', measuresNotFound);
+
+  // separate passes for ecqm data to keep things simple
+  // load extracted ecqm data
+  const generatedEcqmStrataJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../util/generated-ecqm-data.json'), 'utf8'));
```

Comment:
I didn't find anything to add to `measuresNotFound` - I've already added logging for potential issues when adding ecqm data, so I moved the `measuresNotFound` flushing to the bottom to make it clear that it happens at the end.